jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererNotificationTest.kt

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codewhisperer
+
+import com.intellij.internal.statistic.eventLog.FeatureUsageUiEvents
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.testFramework.TestActionEvent
+import com.intellij.testFramework.replaceService
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.spy
+
+class CodeWhispererNotificationTest : CodeWhispererBasicTestBase() {
+    private lateinit var event: AnActionEvent
+    private lateinit var featureUsageUiEventsSpy: FeatureUsageUiEvents
+
+    @Before
+    fun setUp() {
+        featureUsageUiEventsSpy = spy(ApplicationManager.getApplication().getService(FeatureUsageUiEvents::class.java))
+        ApplicationManager.getApplication().replaceService(FeatureUsageUiEvents::class.java, featureUsageUiEventsSpy, disposableRule.disposable)
+        event = TestActionEvent { key ->
+            when (key) {
+                CommonDataKeys.PROJECT.name -> projectRule.project
+                else -> null
+            }
+        }
+    }
+
+    @Test
```

Comment:
Since these tests are commented out, is this class and the open class above required?

Reply:
ah sorry - I will just remove these for now.